examples/webcam.rs

```diff
@@ -0,0 +1,47 @@
+use kornia_rs::{image::ImageSize, io::webcam::WebcamCaptureBuilder};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // start the recording stream
+    let rec = rerun::RecordingStreamBuilder::new("Kornia App")
+        // set the batcher config to flush every 10 bytes to improve performance
+        .batcher_config(rerun::log::DataTableBatcherConfig {
+            flush_num_bytes: 10,
```

Comment:
10bytes is a bit extreme, that effectively disables the batcher, meaning every interaction with the recording stream will right away send to the viewer which can be overall much slower.
The default is to send either every 1mb or ever 8ms. So if the default is still a problem running in release mode it might make actually more sense to lower `flush_tick`.
I guess alternatively you could also make it very explicit and go with `rerun::log::DataTableBatcherConfig::ALWAYS`, that's more honest than 10bytes 😄